src/views/Login/Login.js

```diff
@@ -45,38 +46,50 @@ export default function SignIn() {
         event.preventDefault();
       };
 
-      const handleSubmit = (e) => {
+      const handleSubmit = async (e) => {
         e.preventDefault()
-        const savedUser = JSON.parse(localStorage.getItem('user'));
+        const body = {email:values.email,password:values.password};
+
+
+        dispatch({type: SET_LOADING, payload: true })
         if (values.email === '') {
           dispatch({type: SET_ERROR, payload: 'Email is required'})
           toaster(auth.error, 'error')
           return false
         }
-        else if (values.email !== savedUser.Email) {
-          dispatch({type: SET_ERROR, payload: 'invalid Email or Password'})
-          toaster(auth.error, 'error')
-          return false
-        }
         else if (values.password === '') {
           dispatch({type: SET_ERROR, payload: 'Password is required'})
           toaster(auth.error, 'error')
           return false
-        }
-         else if (values.password !== savedUser.Password) {
-          dispatch({type: SET_ERROR, payload: 'invalid Email or Password'})
-          toaster(auth.error, 'error')
-          return false
-        }
-          dispatch({type: SET_AUTHENTICATION})
-          toaster('You are logged in', 'success')
-          setTimeout(() => {
-            history.push('/dashboard')
-          }, 4000)   
+        } 
+        const response =  await fetch(`${REACT_APP_BACKEND_URL}/user/signin`,
+          {
+            method:'post',
+            headers:{"Content-Type":"Application/json"},
+            body:JSON.stringify(body)
+           });
+           const jsonData = await response.json();
+
+           if(jsonData.user !== undefined){
+               localStorage.setItem("userId",jsonData.user.id);
+               localStorage.setItem("userToken",jsonData.token);
+               localStorage.setItem("userImageUrl",jsonData.user.user_image);
+               dispatch({type: SET_LOADING, payload: false })      
+               dispatch({type: SET_AUTHENTICATION, user:jsonData.user, token:jsonData.token })
+               toaster(jsonData.message, 'success')
```

Comment:
This toaster is not working during successful login.

Reply:
I am not the one who did login integration.